pandas/core/indexes/interval.py

```diff
@@ -571,15 +595,21 @@ def from_tuples(cls, data, closed='right', name=None, copy=False):
             if isna(d):
                 lhs = rhs = np.nan
             else:
-                lhs, rhs = d
+                try:
```

Comment:
can you add a 1-liner here showing the format of th expected input (list-of-tuples)

Reply:
done